src/client/pages/MarkdownPage.tsx

```diff
@@ -807,8 +829,4 @@ MarkdownPage.propTypes = {
   loginCallback: PropTypes.string,
 };
 
-MarkdownPage.defaultProps = {
```

Comment:
Noticed a deprecated warning about this and saw other pages used `loginCallback = '/'` in the component definition, so updated